test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -52,17 +54,24 @@ UcxLog::UcxLog(const char* prefix, bool enable) : _enable(enable)
     struct tm tm;
     char str[32];
     if (use_human_time) {
-        strftime(str, sizeof(str), "[%a %b %d %T] ", localtime_r(&tv.tv_sec, &tm));
+        strftime(str, sizeof(str), "[%a %b %d %T ", localtime_r(&tv.tv_sec, &tm));
     } else {
-        snprintf(str, sizeof(str), "[%lu.%06lu] ", tv.tv_sec, tv.tv_usec);
+        snprintf(str, sizeof(str), "[%lu.%06lu ", tv.tv_sec, tv.tv_usec);
     }
-    std::cout << str << prefix << " ";
+
+    size_t str_len = strlen(str);
+    snprintf(str + str_len, sizeof(str) - str_len, "%d] ", getpid());
```

Comment:
maybe move to the same format print above?

Reply:
you mean call getpid in both places? 'if' and 'else'

Reply:
yes

Reply:
kind of code duplication, calling to getpid twice if can call it once.

Reply:
but no need to `strlen` and one more format print

Reply:
after discussing offline, keeping this as is.